.github/workflows/deployment.yaml

```diff
@@ -12,13 +12,14 @@ jobs:
       matrix:
         architecture:
           - x64
+          - arm64
         os:
           - macos-13
           - ubuntu-22.04
           - windows-2022
-        include:
+        exclude:
           - architecture: arm64
-            os: macos-14
+            os: windows-2022
```

Comment:
Windows ARM64 runners [are also available](https://github.blog/changelog/2024-06-03-actions-arm-based-linux-and-windows-runners-are-now-in-public-beta/). 
Do we also want to support windows?